.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.ts text eol=crlf
+*.js text eol=crlf

```

Comment:
Should change the line break style es lint rule to be on if you have fixed this issue